includes/api/cart.php

```diff
@@ -39,6 +39,10 @@ function fetchCartData(WP_REST_Request $request)
 
     $response = cartResponse($couponCode);
 
+    $response['user'] = getCartUserObject();
```

Comment:
Why do we need this info if the Terra Wallet plugin is not activated on the store? Will it not add some latency to cart fetch/create API?

Reply:
@ChetanGN we need this to fetch store's logged in user email